src/views/clusteroverview/overview/components/top-consumers-card/utils/DurationOption.ts

```diff
@@ -0,0 +1,53 @@
+import DropdownEnum from '@kubevirt-utils/utils/dropdownEnum';
+import { ObjectEnum } from '@kubevirt-utils/utils/ObjectEnum';
+
+class DurationOption extends DropdownEnum<string> {
+  static readonly FIVE_MIN = new DurationOption('5m', {
+    // t('5 minutes')
```

Comment:
I think this should be `// t('plugin__kubevirt-plugin~5 minutes')` and should be moved to src/utils/i18n.ts as well as the other similar comments in this file

Reply:
Why? `yarn i18n` picked them up. Is there something I'm missing?

Reply:
AFAIK src/utils/i18n.ts is to make some file that contains all these kind of comments to the translation parser, not as a strict rule but more of a convention to us :)
and Im almost sure when you put it in a comment like you did, we need to prefix `plugin__kubevirt-plugin~` to the string we want to translate.
adding @glekner to this since he has more knowledge of this than me :)

Reply:
I went ahead and changed it, but further explanation would be helpful so I'll understand the difference. =)

Reply:
no need for prefix